apps/src/craft/craft.js

```diff
@@ -733,6 +734,12 @@ Craft.reportResult = function (success) {
   });
 };
 
+Craft.shouldDefaultToContinue = function(testResultType) {
```

Comment:
Could probably use a function comment giving a little more context, i.e. at the least specifying that this refers to the finish dialog.